tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -954,9 +954,9 @@ install_ovmf() {
 	tar xvf "${builddir}/${tarball_name}" -C "${destdir}"
 }
 
-# Install OVMF SEV
-install_ovmf_sev() {
-	install_ovmf "sev" "edk2-sev.tar.gz"
+# Install OVMF SNP
+install_ovmf_snp() {
+	install_ovmf "snp" "edk2-sev.tar.gz"
```

Comment:
Should the name of the tar file change as well?

Reply:
Also, `edk2-sev.tar.xz` and instructions mentioning SEV are in the docs related to SNP as well.